src/main/java/seedu/meditracker/medication/Medication.java

```diff
@@ -0,0 +1,45 @@
+package seedu.meditracker.medication;
+
+/**
+ * The Medication class represents a medication.
+ * It stores information about the medication such as name, quantity, dosage, expiry date, intake frequency, remarks,
+ * and whether it has been taken.
+ */
+public class Medication {
+
+    // Medication attributes are intentionally declared String. To be modified later.
+    private String medicationName;
+    private String medicineQuantity;
+    private String medicineDosage;
+    private String expiryDate;
+    private String intakeFreq;
+    private String remarks;
+    private boolean isTaken = false;
```

Comment:
isTaken will be tracked separately in a different class, can be removed from this class

```suggestion
```